docs/proposals/cli.md

```diff
@@ -0,0 +1,328 @@
+
+**Arguments :**
+
+`kind` : the kind of the resource, possible values :
+
+- workload
+
+**Flags** :
+
+| Name        | Shorthand | Values | Default | Description                                     |
+| ----------- | --------- | ------ | ------- | ----------------------------------------------- |
+| --file      | -f        | Path   | `""`    | add resource definition from file.              |
+| --no-update |           | bool   | false   | If the resource already exists, don’t update it |
+| --name      |           | string | `""`    | set the name of the resource                    |
+| --help      | -h        |        | false   | show help of the function.                      |
+
+**Kind specific flags :**
+
+*workload :*
+
+| Name             | Shorthand | Values   | Default       | Description                                                                           |
+| ---------------- | --------- | -------- | ------------- | ------------------------------------------------------------------------------------- |
+| --type           |           | string   | `"container"` | workload type                                                                         |
+| --uri            |           | string   | `""`          | the uri                                                                               |
+| --resources-cpu  |           | integer  | 1             | the cpu amount                                                                        |
+| --resources-ram  |           | integer  | 50            | the ram amount      (MB)                                                              |
+| --resources-disk |           | integer  | 1             | the disk size (GB)                                                                    |
+| --port           | -p        | []string | []            | ports binding list, use multiple times to add multiple elements to the array          |
+| --environment    |           | []string | []            | environment variables list, use multiple times to add multiple elements  to the array |
+
+**Examples :**
+
+- Add resource from file
+
+  ```sh
+  kudoctl create resource -f workload.yml
+  ```
+
+</details>
+  
+---
+
+<details> <summary><h3>apply</h3></summary>
+
+This command takes the same arguments as `create <kind>` except the kind is defined by the `--kind` flag , creates a resource, then instanciate it.
+
+**Flags :**
+
+| Name   | Shorthand | Values | Default      | Description                |
+| ------ | --------- | ------ | ------------ | -------------------------- |
+| --kind |           | string | `"workload"` | resource kind              |
+| --help | -h        |        | false        | show help of the function. |
+
+**Example:**
+
+```bash
+kudoctl apply -f workload.yml
+```
+
+</details>
+
+---
+  
+<details> <summary><h3>instantiate &lt;resource-id&gt;</h3></summary>
+
+Instantiate and start a resource
```

Comment:
What does you mean by starting a resource? If a user is a resource, what does starting a user mean?